apps/src/feedback.js

```diff
@@ -448,8 +439,9 @@ FeedbackUtils.prototype.displayFeedback = function (
   const publishButtonSelector = '#publish-to-project-gallery-button';
   const publishButton = feedback.querySelector(publishButtonSelector);
   if (publishButton) {
-    dom.addClickTouchEvent(publishButton, () => {
+    dom.addClickTouchEvent(publishButton, () => { // TODOLOG
```

Comment:
is "TODOLOG" a convention of ours?

Reply:
Haha! No - just a note to myself when I first looked at this file to know where to add logging. Removed!